core/app/models/mno_enterprise/audit_event.rb

```diff
@@ -20,7 +20,7 @@ def formatted_details
     def format_serialized_details
       AUDIT_LOG_CONFIG.fetch('events', {}).fetch(key, '') % metadata.symbolize_keys
     rescue KeyError => e
-      e.message
+      e.message unless key == 'subscription_update'
```

Comment:
Why did you need this?

Reply:
I did this as it was returning the message in the audit log and it looked messy.